NameError: name 'Blueprint' is not defined

Hi!

I wanted to try the project from the "quickstart" but I hit

>NameError: name 'Blueprint' is not defined

After running these steps:

```bash
pip install -e .
mkdir ~/data
mephisto config core.main_data_directory ~/data
mephisto check
# OK
cd examples/simple_static_task
python static_test_script.py
```

Any idea? Seems to be related to Flask.

+1

pip install flask does not resolve

looks like requirements might not be checked in the right order?

here is the trace

python3 static_test_script.py
/opt/homebrew/lib/python3.8/site-packages/hydra/_internal/defaults_list.py:251: UserWarning: In 'scriptconfig': Defaults list is missing `_self_`. See https://hydra.cc/docs/upgrades/1.0_to_1.1/default_composition_order for more information
  warnings.warn(msg, UserWarning)
[2021-11-01 12:50:33,586][mephisto.operations.operator][ERROR] - Ran into error while launching run: 
Traceback (most recent call last):
  File "/Users/potamian/Mephisto/mephisto/operations/operator.py", line 429, in validate_and_run_config
    return self.validate_and_run_config_or_die(
  File "/Users/potamian/Mephisto/mephisto/operations/operator.py", line 169, in validate_and_run_config_or_die
    BlueprintClass.assert_task_args(run_config, shared_state)
  File "/Users/potamian/Mephisto/mephisto/abstractions/blueprints/static_html_task/static_html_blueprint.py", line 110, in assert_task_args
    Blueprint.assert_task_args(args, shared_state)
NameError: name 'Blueprint' is not defined
[2021-11-01 12:50:33,587][mephisto.operations.operator][INFO] - operator shutting down

--- a/mephisto/abstractions/blueprints/static_html_task/static_html_blueprint.py
+++ b/mephisto/abstractions/blueprints/static_html_task/static_html_blueprint.py
@@ -10,6 +10,7 @@ from mephisto.abstractions.blueprints.abstract.static_task.static_blueprint impo
 )
 from dataclasses import dataclass, field
 from omegaconf import MISSING, DictConfig
+from mephisto.abstractions.blueprint import Blueprint
 from mephisto.abstractions.blueprints.static_html_task.static_html_task_builder import (
     StaticHTMLTaskBuilder,
 )

